src/keras_geometric/datasets/base.py

```
@@ -122,14 +122,18 @@ def _process_raw(self) -> tuple[list[GraphData], int]:
         return data_list, num_classes
 
     def _save_processed(self):
-        """Save processed data to disk."""
+        """
+        Saves the processed dataset to disk in NumPy `.npz` format.
+        
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)